server/src/metrics/edge_metrics.rs

```diff
@@ -310,10 +310,10 @@ pub struct EdgeInstanceData {
 }
 
 impl EdgeInstanceData {
-    pub fn new(app_name: &str) -> Self {
+    pub fn new(app_name: &str, identifier: &Ulid) -> Self {
```

Comment:
Let's us define an instance ID in the main method and pass it in, rather than trying to get it off the struct later on